src/details/ArborX_DetailsUnionFind.hpp

```diff
@@ -171,37 +146,28 @@ struct UnionFind
     // until there is no data race on the parent.
     // ```
 
-    // ##### ECL license (see LICENSE.ECL) #####
```

Comment:
Please confirm you meant to remove this line.

Reply:
Confirm.